docs/user/courses/customizable.rst

```diff
@@ -0,0 +1,76 @@
+
+Administrators can then specify options in the creation form, instructors can later change these values:
+
+- **Title**: Name of the course that is shown to users e.g. in the |course_overview|. Can be changed later.
+- **Short Name**: Unique identifier of the course. Can **not** be changed later as it is included in resources of external systems such as build plans.
+- **Color**: Color of the course that will be used e.g. in the |course_overview|.
+- **Course Description**: Description of the course that will e.g. be shown in the Course Registration table and in the header of the course (once a user clicks on the course in their dashboard).
+- **Course Organizations**: Users and Courses can be grouped into different organizations (e.g. based on the email-address of users). Users will then only see courses from their organizations, allowing multiple organizations to use the same Artemis environment without interfering with each other.
+  If no course organization is set, all users can interact with the course.
+- **Start/End**: Start and End Date of the course. Users can only self-register to a course between the Start and End Date.
+- **Test Course**: Whether this course should be used for testing. Upcoming exams and exercises within Test Courses are not displayed to users.
+- **Semester**: The semester in which the course is conducted. Courses are grouped by semesters to simplify the search in Artemis environments with a large number of courses.
+- **Maximum number of points for course**: This number is used for the grading key calculation. Updating the maximum number will recalculate the grading key, if it has been set up before.
+- **Amount of decimal places used for calculating the scores**: Specify the granularity for score calculations. You might have to increase this value if you create exercises with a small number of maximum points and a large number of tests.
+- **Default Programming Language**: The programming language that should be preselected when a new programming exercise is created in this course.
+- **Customize group names**: Allows specifying custom group names that should be used in external user management systems. Default group names (based on the course short name) will be used if not specified. **Note**: Only administrators can change this setting. Instructors can not change it.
+- **Enable complaints**: Whether complaints should be enabled in this course. Details regarding complaints can be found `here </user/exercises/general#complaints>`_.
+- **Enable more feedback requests**: Whether more feedback requests should be enabled in this course. Details regarding more feedback requests can be found `here </user/exercises/general#more-feedback-requests>`_.
+- **Enable postings by students**: Whether the posting sections should be enabled for lectures and exercises. More details regarding this can be found in the `here </user/communication>`_.
```

Comment:
```suggestion
- **Enable postings by students**: Whether the posting sections should be enabled for lectures and exercises. More details regarding this can be found `here </user/communication>`_.
```